### PR TITLE
Bulk actions

### PR DESCRIPTION
**Pull request by ftomassetti**
Initial implementation of bulk actions both for the project page and the views by patch type.

Looking for initial feedback.

### CODE REVIEW

File: app/templates/base.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
         <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/bootstrap.css')}}" />
         <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/bootstrap-theme.css')}}" />
         <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/plaid.css')}}" />
+        <script src="{{ url_for('static', filename='js/jquery-2.0.3.js') }}"></script>
```
Comment (lu-zero) [inline app/templates/base.html]: It is already at the end of the file.

Comment (ftomassetti) [inline app/templates/base.html]: left the includes only at the top of the file

**Comment by ftomassetti** [comment]
Added a check on the current user. Users which are not committers or admins do not see at all the checkboxes and controls to perform the bulk action

**Comment by ftomassetti** [comment]
Note to myself: use $selector.click(callback) and start from the branch prepared by Luca

**Comment by lu-zero** [comment]
Cherry-picked and fixed.

**Pull request closed without merging**